### PR TITLE
Add link to FEEL temporal expression examples

### PR DESCRIPTION
**Pull request by falko**
## Description

Link to [very helpful examples from FEEL Scala docs](https://camunda.github.io/feel-scala/docs/learn/samples/temporal-samples)

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
I'd like to see a little more context around this link, but we should probably also backport it, maybe to all supported versions. @camunda/tech-writers can one of you assist Falko with both of those things? 🙂

**Comment by mesellings** [comment]
@falko I'll take a look and backport as well for you 👍

**Comment by saig0** [comment]
@falko instead of adding a link to the FEEL-Scala docs, I would prefer adding the examples directly in the C8 docs. On a long-term, I would move all relevant content to the Camunda docs (and the code itself to the Camunda repo).

**Comment by mesellings** [comment]
@saig0 @falko I think moving content into the docs may be quite a lift for this PR, and probably something we need to plan out - there are issues such as redirects/links to the github docs for example that we need to take into consideration.

Can I suggest the following plan:

For now (for this PR) we add the links in as suggested, and we can reference the examples a bit more clearly. For example:

> **Info:** Temporal expression examples are available in the [FEEL-Scala documentation](https://camunda.github.io/feel-scala/docs/learn/samples/temporal-samples), including how to compare and check dates, calculate age and duration, and change date format.

Then, we address moving relevant docs over (such as these examples) as part of a larger project?

**Comment by falko** [comment]
@akeller can we just merge all examples from here: https://github.com/camunda/feel-scala/tree/main/docs/docs/learn/samples

Really helpfule, would be to somehow integrate the FEEL playground into the docs so that users can directly modify the examples to match their use cases. It could be done as a link near each example or even directly like how @saig0 did it in his [FEEL Challenge](https://camunda.github.io/feel-scala/docs/learn/challenge/chapter-1). Having the examples in the playground could also be a way to maintain them, i.e. test that they still work.

**Comment by akeller** [comment]
> @akeller can we just merge all examples from here: https://github.com/camunda/feel-scala/tree/main/docs/docs/learn/samples
> 
> Really helpfule, would be to somehow integrate the FEEL playground into the docs so that users can directly modify the examples to match their use cases. It could be done as a link near each example or even directly like how @saig0 did it in his [FEEL Challenge](https://camunda.github.io/feel-scala/docs/learn/challenge/chapter-1). Having the examples in the playground could also be a way to maintain them, i.e. test that they still work.

Let's handle integrating the playground in a separate issue/PR scope so we can pull in @pepopowitz to incorporate this. @pepopowitz can you sync with @saig0 and create an issue for this? 

We can merge the examples/samples into the C8 docs, but I want to be clear DevEx wouldn't update them in both places. So within the scope of this PR, instead of linking, we would add list, context, and temporal expression examples to the existing pages in the C8 docs. @mesellings can you rework this PR? Or open a new one and add @falko and @saig0 as reviewers?

**Comment by mesellings** [comment]
@falko @saig0 Please review - I have added the examples into the context, temporal, and list expressions pages. and edited for conciseness, typos, and structure. If you are happy with these changes, I will backport for other doc versions (8.6 > 8.2).

**Note:** As mentioned:
* These examples are now branched off from the [FEEL-Scala documentation](https://camunda.github.io/feel-scala/docs/learn/samples/temporal-samples) and we will only update these in the Camunda 8 documentation. 
* The playground integration is being investigated by @pepopowitz and @saig0. It is not included in the scope of this PR.

**Comment by mesellings** [comment]
Note, once these are approved by Engineering, I will add @camunda/tech-writers for review.

File: docs/components/modeler/feel/language-guide/feel-temporal-expressions.md
Comment (saig0) [inline docs/components/modeler/feel/language-guide/feel-temporal-expressions.md:484]: :wrench: Simplify expression.

```suggestion
{
  time: @"09:00:00@Europe/Berlin",
  date: if time(now()) < time then today() else today() + @"P1D",
  duration: date and time(date, time) - now()
}.duration
```

File: docs/components/modeler/feel/language-guide/feel-temporal-expressions.md
Comment (saig0) [inline docs/components/modeler/feel/language-guide/feel-temporal-expressions.md:495]: :wrench: Simplify expression.

```suggestion
(for x in 1..3
  return date and time(today(),@"00:00:00Z") + @"P1D" * x
)[not(day of week(item) in ("Saturday","Sunday"))][1]
```

File: docs/components/modeler/feel/language-guide/feel-temporal-expressions.md
Comment (saig0) [inline docs/components/modeler/feel/language-guide/feel-temporal-expressions.md:529]: :wrench: Simplify expression.

```suggestion
(now() - @"1970-01-01T00:00Z") / @"PT1S" * 1000
```

**Review by saig0 — CHANGES_REQUESTED** [review]
@mesellings thank you for adding the examples. :+1: 

I added a few comments. Please have a look. :cookie:

File: docs/components/modeler/feel/language-guide/feel-context-expressions.md
Comment (mesellings) [reply docs/components/modeler/feel/language-guide/feel-context-expressions.md]: Agreed, changed for all 3 pages

**Review by mesellings — DISMISSED** [review]
@falko @saig0 I've committed the comments and made changes from @saig0 review, I'm happy to approve this PR now @falko  if you are happy with the code suggestions applied from the @saig0 review 👍

File: docs/components/modeler/feel/language-guide/feel-list-expressions.md
Comment (saig0) [inline docs/components/modeler/feel/language-guide/feel-list-expressions.md:183]: :wrench: Remove semicolon.

```suggestion
]
```

File: docs/components/modeler/feel/language-guide/feel-list-expressions.md
Comment (saig0) [inline docs/components/modeler/feel/language-guide/feel-list-expressions.md:222]: :wrench: Remove semicolon.

```suggestion
]
```

File: docs/components/modeler/feel/language-guide/feel-temporal-expressions.md
Comment (saig0) [inline docs/components/modeler/feel/language-guide/feel-temporal-expressions.md:511]: :wrench: Remove semicolon.

```suggestion
["2021-04-21T07:25:06.000Z", "2021-04-22T07:25:06.000Z"]
```

File: docs/components/modeler/feel/language-guide/feel-temporal-expressions.md
Comment (saig0) [inline docs/components/modeler/feel/language-guide/feel-temporal-expressions.md:517]: :wrench: Remove semicolon.

```suggestion
["21-Apr-2021", "22-Apr-2021"]
```

File: docs/components/modeler/feel/language-guide/feel-temporal-expressions.md
Comment (saig0) [inline docs/components/modeler/feel/language-guide/feel-temporal-expressions.md:531]: :wrench: Remove semicolon.

```suggestion
1618200039000
```

**Review by saig0 — APPROVED** [review]
@mesellings great work. LGTM. :tada:

**Comment by mesellings** [comment]
@falko Do you want me to merge and release this work?

**Review by mesellings — APPROVED** [review]
Lgtm 👍 🚀

**Comment by mesellings** [comment]
@falko I've merged for you and I'll do a docs release this week to get the work out 👍

**Pull request merged**
